mono/mini/TestDriver.cs

```diff
@@ -13,10 +13,40 @@ public class CategoryAttribute : Attribute
 		get; set;
 	}
 }
+public class TestDriverReporter
+{
+	private int failedTests = 0;
+	private int skippedTests = 0;
+	private int executedTests = 0;
+
+	public int FailedTests {
+		get {
+			return failedTests;
+		}
+	}
+
+	public int SkippedTests {
+		get {
+			return skippedTests;
+		}
+	}
+
+	public int ExecutedTests {
+		get {
+			return executedTests;
+		}
+	}
```

Comment:
I did it this way since the field and the property accessing the field was intentionally split so the assignment in ReportResults could write directly to the underlying field without the need to go through a property to do so. I have no problems simplify it, hopefully the release build will optimize the assignments, not doing a call but a mov. A mov will always be used with current construction.